### ISSUE TITLE
CPU Voltage stucked at 865mv even with 2397mhz OC at full load

### ISSUE THREAD

**Issue by willowdale16** [KazushiMe/Switch-OC-Suite#30]
so i just tested max cpu OC on some cpu bound games with 60fps patch and memtester. The voltage on the CPU stucked at 865mv for some reason. it never go higher than that, which mean the performance feel like im using 1900mhz oc. i already turned of the allow unsafe freq.

**Comment by KazushiMe** [comment]
Check your CPU speedo in hekate HW Info and try CPU single core benchmark using [this](https://github.com/KazushiMe/Switch-OC-Suite/files/9751276/CPU_Perf_Test.zip) (disable sys-clk first).

**Comment by Willowdale** [comment]
> Check your CPU speedo in hekate HW Info and try CPU single core benchmark using [this](https://github.com/KazushiMe/Switch-OC-Suite/files/9751276/CPU_Perf_Test.zip) (disable sys-clk first).

![IMG_20221011_121852](https://user-images.githubusercontent.com/106679989/195003655-7d65093d-dfc7-4e3f-a4ab-2983dd3c2765.jpg)

I already tested with custom loader.kip from someone and it stayed at 1220mv all the time with a quite noticable performance gain (around 10%) at same clockspeed 2397 cpu/1996mhz ram. But the kip from here is just stayed at 865mv. 

So i tested you kip again and it reaches 1220mv for the first run in memory tester, but the second run it getting lowered again to 865mv. So it only reached 1220mv for a lil bit. And it stays at 865mv while playing games after that. Already restart and re do the installation and no changes.

**Comment by KazushiMe** [comment]
> I already tested with custom loader.kip from someone and it stayed at 1220mv all the time with a quite noticable performance gain (around 10%) at same clockspeed 2397 cpu/1996mhz ram.

> So i tested you kip again and it reaches 1220mv for the first run in memory tester, but the second run it getting lowered again to 865mv. So it only reached 1220mv for a lil bit.

They might have changed CPU CVB table here https://github.com/KazushiMe/Switch-OC-Suite/blob/c8587ce4410cec851de18c06da7584a6e7234b56/Source/Atmosphere/stratosphere/loader/source/oc/ldr_oc_suite.cpp#L75-L99
Your CPU speedo value seems fine, not sure how the default table won't work.

**Comment by timunkun** [comment]
Yeah have no idea. So i just leave my console for 3 hours and decided to run tinymem again. First run it started at 1055mv (not even 1220) and then the second run, it started to lower the voltage to 865mv again and it stays like that until i leave my switch on sleep for a couple hours. 

This kind of thing isn't a problem with the custom kip that someone gave it to me. It stays at 1220 on 2100mhz above all the time (tested for 5 hours'ish). 

Any fixes or is it happen to you either ? Probably have to wait until ams compatible with 15.0

**Comment by timunkun** [comment]
Any plans for the fixes with the newest AMS release later ?

**Comment by timunkun** [comment]
> > I already tested with custom loader.kip from someone and it stayed at 1220mv all the time with a quite noticable performance gain (around 10%) at same clockspeed 2397 cpu/1996mhz ram.
> 
> > So i tested you kip again and it reaches 1220mv for the first run in memory tester, but the second run it getting lowered again to 865mv. So it only reached 1220mv for a lil bit.
> 
> They might have changed CPU CVB table here
> 
> https://github.com/KazushiMe/Switch-OC-Suite/blob/c8587ce4410cec851de18c06da7584a6e7234b56/Source/Atmosphere/stratosphere/loader/source/oc/ldr_oc_suite.cpp#L75-L99
> 
> 
> Your CPU speedo value seems fine, not sure how the default table won't work.

So the CPU Governor is actually help keeping the voltage at 1220mhz on "High Load" but sometimes the cpu is lowered to 600 even it needs more than that. So is it possible to have a Min and Max CPU Governor? Thank you for your hardwork!

Oh anyway there's a bug on cpu Governor setting, it made the freq stucked at 1000mhz permanently when i change global freq. It back to normal when i restarted the switch.

**Comment by KazushiMe** [comment]
Mariko CPU CVB table has been updated. See if [1.4.0.1-pre](https://github.com/KazushiMe/Switch-OC-Suite/releases/tag/1.4.0.1) works.

**Comment by timunkun** [comment]
> Mariko CPU CVB table has been updated. See if [1.4.0.1-pre](https://github.com/KazushiMe/Switch-OC-Suite/releases/tag/1.4.0.1) works.

![P_20221231_232738](https://user-images.githubusercontent.com/106679989/210149825-4b41b83e-9d17-4a2d-a141-cfa8c1096911.jpg)


The voltage still doing the same like before (without enabling governor)

**Comment by timunkun** [comment]
And with governor enabled. The voltage can be maxed out like usual. But i noticed that the voltage is increased to 1235mv, and apparently the performance on tinymembench is 3% lower compared to the previous kip (5860MB - 5888MB compared to the last kip that has constant 5914MB 5 runs in a row)

**Comment by timunkun** [comment]
![Screenshot_20221231-234528050 (1)](https://user-images.githubusercontent.com/106679989/210150276-b762d4f2-58b7-4806-b7ea-53df27439a9e.png)

Apparently someone got this kind of shenanigans too

**Comment by timunkun** [comment]
> Mariko CPU CVB table has been updated. See if [1.4.0.1-pre](https://github.com/KazushiMe/Switch-OC-Suite/releases/tag/1.4.0.1) works.

Anyway, is the new loader safe to use ? I mean it has higher voltage compared to the old  one. Is it gonna heat more or it runs the same ?
